test/taoensso/faraday/tests/requests.clj

```diff
@@ -219,6 +233,13 @@
     (expect "global-secondary" (.getIndexName action))
     ))
 
+(let [req ^UpdateTableRequest (update-table-request
```

Comment:
Attaching the type-hint to either the symbol or the form makes the reflection-warnings go away.